test/widgets/subscription_list_test.dart

```diff
@@ -132,6 +132,18 @@ void main() {
       ]);
       check(listedStreamIds(tester)).deepEquals([1, 2, 3, 4, 5, 6]);
     });
+
+    testWidgets('muted subscriptions come at last', (tester) async {
```

Comment:
At first I had trouble connecting this test description to the test's behavior: there are three muted subscriptions, for stream IDs 1, 3, and 5, but those IDs don't all come last in the list of expected stream IDs.

I think we mean something like: 'muted subscriptions come last among pinned streams and among unpinned streams'. There might be a more concise way to say that.